<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Saulo Chaves </title> <meta name="author" content="Saulo Chaves"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%AC&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://saulo-chaves.github.io/"> <script src="/assets/js/theme.js?a81d82887dd692e91686b43de4542f18"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">About <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">Publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">Repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">CV </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title"> Saulo <span class="font-weight-bold">Chaves</span> </h1> <p class="desc"></p> </header> <article> <div class="profile float-right"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/1709985435295-480.webp 480w,/assets/img/1709985435295-800.webp 800w,/assets/img/1709985435295-1400.webp 1400w," type="image/webp" sizes="(min-width: 930px) 270.0px, (min-width: 576px) 30vw, 95vw"> <img src="/assets/img/1709985435295.jpeg?5701f204cdbb384e6ef074eeee8068ea" class="img-fluid z-depth-1 rounded" width="100%" height="auto" alt="1709985435295.jpeg" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> <div class="more-info"> <p>Department of Genetics</p> <p>Av. P√°dua Dias, 11</p> <p>Piracicaba, S√£o Paulo, Brazil</p> </div> </div> <div class="clearfix"> <p>Proudly, Brazilian and Amazonian (<strong><a href="https://www.google.com/maps/place/Bel%C3%A9m,+PA/@-1.3413464,-48.611677,11z/data=!3m1!4b1!4m6!3m5!1s0x92a461af84756ce1:0x570d540215864c35!8m2!3d-1.4563432!4d-48.501299!16s%2Fg%2F1pxyyzx3r?entry=ttu&amp;g_ep=EgoyMDI1MDQxNC4xIKXMDSoASAFQAw%3D%3D" rel="external nofollow noopener" target="_blank">Bel√©m - PA</a></strong>). Agronomist and Ph.D. in Plant Breeding. Assistant Professor in the <strong><a href="http://www.genetica.esalq.usp.br/en/" rel="external nofollow noopener" target="_blank">Department of Genetics</a></strong> of the <strong><a href="https://en.esalq.usp.br/" rel="external nofollow noopener" target="_blank">‚ÄúLuiz de Queiroz‚Äù College of Agriculture</a></strong>, <strong><a href="https://www5.usp.br/" rel="external nofollow noopener" target="_blank">University of S√£o Paulo</a></strong>. Leader of the <strong>GAB</strong> (<strong>G</strong>enomics, <strong>A</strong>nalytics and <strong>B</strong>reeding) lab.</p> <p>I‚Äôm very interested in plant breeding, statistical and quantitative genetics and solutions to integrate data science into plant breeding routines. I also love football and music (progressive rock/metal rules! ü§òüèΩ). Feel free to contact me if you got interested in any of my research fields, or you‚Äôre feeling like talking to someone!</p> </div> <h2> <a href="/publications/" style="color: inherit">Selected publications</a> </h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/cupu.JPG" class="preview z-depth-1 rounded" width="100%" height="auto" alt="cupu.JPG" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="chaves_analysis_2023" class="col-sm-8"> <div class="title">Analysis of repeated measures data through mixed models: An application in Theobroma grandiflorum breeding</div> <div class="author"> <em>Saulo Chaves</em>, Rodrigo S. Alves, Luiz A. S. Dias, Rafael M. Alves, Kaio O. G. Dias, and Jeniffer S. P. C. Evangelista </div> <div class="periodical"> <em>Crop Science</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1002/csc2.20995" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.1002/csc2.20995"></span> <span class="__dimensions_badge_embed__" data-doi="10.1002/csc2.20995" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Theobroma grandiflorum is a perennial fruit tree native to the Amazon region. As a perennial species with continuous production throughout the years, breeders should seek well-conducted trials, accurate phenotyping and adequate statistical models for genetic evaluation and selection that can leverage the information provided by the repeated measures. We evaluated 13 models with different covariance structures for genetic and residual effects for T. grandiflorum evaluation, using an unbalanced dataset with 34 hybrids from the triple-crossing of nine parents, planted in a randomized complete block design. For nine consecutive years, the fruit yield of these hybrids was evaluated. Each model had its goodness-of-fit tested by the Akaike information criterion. The most adequate model for estimating the variance components and the breeding values were modelled with the first-order heterogeneous autoregressive for residual effects and third-order factor analytic for genetic effects. From this model, we used the factor analytic selection tools for selecting the top 10 families, providing a genetic gain of 10.42%. These results are important not only for T. grandiflorum breeding but also to show that in any repeated measures‚Äô data from fruit-bearing perennial species the modelling of genetic and residual effects should not be neglected.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">chaves_analysis_2023</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Analysis of repeated measures data through mixed models: {An} application in {{Theobroma} grandiflorum} breeding}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{63}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{0011-183X, 1435-0653}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{Analysis of repeated measures data through mixed models}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1002/csc2.20995}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Crop Science}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chaves, Saulo and Alves, Rodrigo S. and Dias, Luiz A. S. and Alves, Rafael M. and Dias, Kaio O. G. and Evangelista, Jeniffer S. P. C.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2131--2144}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/probbreed.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="probbreed.png" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="chaves_probbreed_2024" class="col-sm-8"> <div class="title">ProbBreed: a novel tool for calculating the risk of cultivar recommendation in multienvironment trials</div> <div class="author"> <em>Saulo Chaves</em>, Matheus D. Krause, Luiz A. S. Dias, Antonio A. F. Garcia, and Kaio O. G. Dias </div> <div class="periodical"> <em>G3 Genes|Genomes|Genetics</em>, Mar 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/g3journal/jkae013" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.1093/g3journal/jkae013"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/g3journal/jkae013" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Neglecting genotype-by-environment interactions in multienvironment trials (MET) increases the risk of flawed cultivar recommendations for growers. Recent advancements in probability theory coupled with cutting-edge software offer a more streamlined decision-making process for selecting suitable candidates across diverse environments. Here, we present the user-friendly ProbBreed package in R, which allows breeders to calculate the probability of a given genotype outperforming competitors under a Bayesian framework. This article outlines the package‚Äôs basic workflow and highlights its key features, ranging from MET model fitting to estimating the per se and pairwise probabilities of superior performance and stability for selection candidates. Remarkably, only the selection intensity is required to compute these probabilities. By democratizing this complex yet efficient methodology, ProbBreed aims to enhance decision-making and ultimately contribute to more accurate cultivar recommendations in breeding programs.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">chaves_probbreed_2024</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{ProbBreed}: a novel tool for calculating the risk of cultivar recommendation in multienvironment trials}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2160-1836}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{{ProbBreed}}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1093/g3journal/jkae013}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/g3journal/jkae013}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2024-03-12}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{G3 Genes|Genomes|Genetics}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chaves, Saulo and Krause, Matheus D. and Dias, Luiz A. S. and Garcia, Antonio A. F. and Dias, Kaio O. G.}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">mar</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{jkae013}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/gisfa.PNG" class="preview z-depth-1 rounded" width="100%" height="auto" alt="gisfa.PNG" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="araujo_gis-fa_2024" class="col-sm-8"> <div class="title">GIS-FA: an approach to integrating thematic maps, factor-analytic, and envirotyping for cultivar targeting</div> <div class="author"> Maur√≠cio S. Ara√∫jo, <em>Saulo Chaves</em>, Luiz A. S. Dias, Filipe M. Ferreira, Guilherme R. Pereira, Andr√© R. G. Bezerra, Rodrigo S. Alves, Alexandre B. Heinemann, Fl√°vio Breseghello, Pedro C. S. Carneiro, Matheus D. Krause, Germano Costa-Neto, and Kaio O. G. Dias </div> <div class="periodical"> <em>Theoretical and Applied Genetics</em>, Mar 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1007/s00122-024-04579-z" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.1007/s00122-024-04579-z"></span> <span class="__dimensions_badge_embed__" data-doi="10.1007/s00122-024-04579-z" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>We propose an ‚Äúenviromics‚Äù prediction model for recommending cultivars based on thematic maps aimed at decision-makers.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">araujo_gis-fa_2024</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{GIS}-{FA}: an approach to integrating thematic maps, factor-analytic, and envirotyping for cultivar targeting}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{137}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1432-2242}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{{GIS}-{FA}}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s00122-024-04579-z}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2024-03-12}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Theoretical and Applied Genetics}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Ara√∫jo, Maur√≠cio S. and Chaves, Saulo and Dias, Luiz A. S. and Ferreira, Filipe M. and Pereira, Guilherme R. and Bezerra, Andr√© R. G. and Alves, Rodrigo S. and Heinemann, Alexandre B. and Breseghello, Fl√°vio and Carneiro, Pedro C. S. and Krause, Matheus D. and Costa-Neto, Germano and Dias, Kaio O. G.}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">mar</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{80}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/macauba.jpeg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="macauba.jpeg" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="couto_training_2024" class="col-sm-8"> <div class="title">Training set optimization is a feasible alternative for perennial orphan crop domestication and germplasm management: an Acrocomia aculeata example</div> <div class="author"> Evellyn G. O. Couto, <em>Saulo Chaves</em>, Kaio Olimpio G. Dias, Jonathan A Morales-Marroqu, Alessandro Alves-Pereira, Carlos Augusto Colombo, and Maria Imaculada Zucchi </div> <div class="periodical"> <em>Frontiers in Plant Science</em>, Mar 2024 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.3389/fpls.2024.1441683" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fpls.2024.1441683"></span> <span class="__dimensions_badge_embed__" data-doi="10.3389/fpls.2024.1441683" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">couto_training_2024</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Training set optimization is a feasible alternative for perennial orphan crop domestication and germplasm management: an {{Acrocomia} aculeata} example}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{15}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{All rights reserved}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fpls.2024.1441683}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Plant Science}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Couto, Evellyn G. O. and Chaves, Saulo and Dias, Kaio Olimpio G. and Morales-Marroqu, Jonathan A and Alves-Pereira, Alessandro and Colombo, Carlos Augusto and Zucchi, Maria Imaculada}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1441683}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/euca.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="euca.jpg" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="chaves_realized_2024" class="col-sm-8"> <div class="title">Realized genetic gain with reciprocal recurrent selection in a Eucalyptus breeding program</div> <div class="author"> <em>Saulo Chaves</em>, Luiz A. S. Dias, Rodrigo S. Alves, Filipe M. Ferreira, Maur√≠cio S. Ara√∫jo, Marcos D. V. Resende, Elizabete K. Takahashi, Jo√£o E. Souza, Fernando P. Leite, Samuel B. Fernandes, and Kaio Olimpio G. Dias </div> <div class="periodical"> <em>Tree Genetics &amp; Genomes</em>, Mar 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1007/s11295-024-01678-2" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.1007/s11295-024-01678-2"></span> <span class="__dimensions_badge_embed__" data-doi="10.1007/s11295-024-01678-2" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Key message: Eucalyptus breeding can beneÔ¨Åt from strategies that capture dominance effects, as shown by the improvement in mean annual increment of wood volume across cycles of RRS. Abstract: There is no empirical validation of reciprocal recurrent selection (RRS) in Eucalyptus breeding. Our study helps to Ô¨Åll this gap by quantifying the realized response to selection achieved after two cycles of RRS involving Eucalyptus urophylla and E. grandis. We also investigated the selection effects on the genetic parameters of the breeding populations. We evaluated 25 trials of the Ô¨Årst cycle (C1) of RRS and 12 trials of the second cycle (C2) of RRS. These trials were established in two different regions, separated according to altitude. Fitting linear mixed models enabled the estimation of variance components and the prediction of mean components (general and speciÔ¨Åc hybridizing abilities). The realized response to selection was calculated as the difference between the mean of the predicted genotypic values of the C1 and C2. The RRS effectively improved the mean annual increment of wood volume by 28.5% in the high-altitude region and 12.3% in the low-altitude region from the C1 to C2. The genetic variability also increased as a result of the new genotypes that arose through recombination. These Ô¨Åndings provide insights for decision-making and reinforce that Eucalyptus breeding can beneÔ¨Åt from strategies that capture dominance effects.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">chaves_realized_2024</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Realized genetic gain with reciprocal recurrent selection in a {{Eucalyptus}} breeding program}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{20}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{All rights reserved}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s11295-024-01678-2}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Tree Genetics \&amp; Genomes}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chaves, Saulo and Dias, Luiz A. S. and Alves, Rodrigo S. and Ferreira, Filipe M. and Ara√∫jo, Maur√≠cio S. and Resende, Marcos D. V. and Takahashi, Elizabete K. and Souza, Jo√£o E. and Leite, Fernando P. and Fernandes, Samuel B. and Dias, Kaio Olimpio G.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{47}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/gencomp.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="gencomp.png" data-zoomable loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="chaves_incorporating_2025" class="col-sm-8"> <div class="title">Incorporating spatial and genetic competition into breeding pipelines with the R package gencomp</div> <div class="author"> <em>Saulo Chaves</em>, Filipe M. Ferreira, Getulio C. Ferreira, Salvador A. Gezan, and Kaio Olimpio G. Dias </div> <div class="periodical"> <em>Heredity</em>, Jan 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41437-024-00743-9" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41437-024-00743-9"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41437-024-00743-9" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Genetic competition can obscure the true merit of selection candidates, potentially leading to altered genotype rankings and a divergence between expected and actual genetic gains. Despite a wealth of literature on genetic competition in plant and animal breeding, the separation of genetic values into direct genetic effects (DGE, related to a genotype‚Äôs merit) and indirect genetic effects (IGE, related to the effects of a genotype‚Äôs alleles on its neighbor‚Äôs phenotype) in linear mixed models is often overlooked, likely due to the complexity involved. To address this, we introduce gencomp, a new R package designed to simplify the use of (spatial-) genetic competition models in crop and tree breeding routines. gencomp includes functions for constructing the genetic competition matrix, fitting (spatial-) genetic competition models via the variance-component approach, and extracting key results such as variance components, heritabilities, competition classes, and total genetic values. For tree breeding, gencomp also calculates the merit of different clonal mixtures using the estimated DGE and IGE of the selection candidates. In this paper, we first present the theoretical foundation of the methods implemented in the package. We then demonstrate the use of gencomp with two datasets: one simulated from a Eucalyptus spp. trial and a real potato dataset. We used both datasets to demonstrate the influence of genetic competition in variance component estimates, heritabilities and selection. Despite the dependency on ASReml-R, a paid resource, gencomp is a user-friendly tool that can popularize genetic competition models, contributing to more informed decision-making in plant breeding.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">chaves_incorporating_2025</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Incorporating spatial and genetic competition into breeding pipelines with the {R} package gencomp}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{134}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{2025 The Author(s), under exclusive licence to The Genetics Society}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1365-2540}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41437-024-00743-9}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2025-01-16}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Heredity}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chaves, Saulo and Ferreira, Filipe M. and Ferreira, Getulio C. and Gezan, Salvador A. and Dias, Kaio Olimpio G.}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jan</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Plant breeding}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{129--141}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> </div> <div class="social"> <div class="contact-icons"> <a href="mailto:%73%61%75%6C%6F%63%68%61%76%65%73@%75%73%70.%62%72" title="email"><i class="fa-solid fa-envelope"></i></a> <a href="https://github.com/saulo-chaves" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-github"></i></a> <a href="http://lattes.cnpq.br/7323802421710943" target="_blank" title="Lattes" rel="external nofollow noopener"><i class="ai ai-lattes"></i></a> <a href="https://www.linkedin.com/in/saulo-chaves-752374147" title="LinkedIn" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-linkedin"></i></a> <a href="https://orcid.org/0000-0002-0694-1798" title="ORCID" rel="external nofollow noopener" target="_blank"><i class="ai ai-orcid"></i></a> <a href="https://www.researchgate.net/profile/https://www.researchgate.net/profile/Saulo-Chaves?ev=hdr_xprf/" title="ResearchGate" rel="external nofollow noopener" target="_blank"><i class="ai ai-researchgate"></i></a> <a href="/feed.xml" title="RSS Feed"><i class="fa-solid fa-square-rss"></i></a> <a href="https://scholar.google.com/citations?user=Rt_v_ooAAAAJ&amp;hl" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a> </div> <div class="contact-note"></div> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> ¬© Copyright 2025 Saulo Chaves. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> </body> </html>